README.md

```diff
@@ -44,20 +44,6 @@ To get a Radar publishable API key, [sign up for a Radar account](https://radar.
 
 ### Enable Places
 
-If you are using Places, you must set Facebook as your place data provider.
-
-To set Facebook as your place data provider, call:
-
-```javascript
-Radar.setPlacesProvider(Radar.PLACES_PROVIDER.FACEBOOK);
-```
-
-To disable Places later, call:
-
-```javascript
-Radar.setPlacesProvider(Radar.PLACES_PROVIDER.NONE);
-```
-
 Learn more about [Places](https://radar.io/documentation/places).
 
 ### Identify user
```

Comment:
Looking really good - my main thought is just making the new apis visible to users, can we add some of the calls to the readme? Right now it only talks about trackOnce

Reply:
@corypisano @jsani-radar I'd actually like to start moving a lot of this into Contentful so we can host it on `https://radar.io/documentation` and make changes without pushing to GitHub. I'm happy to own docs.